packages/ngtools/webpack/package.json

```diff
@@ -31,11 +31,10 @@
     "webpack-sources": "^1.1.0"
   },
   "peerDependencies": {
-    "typescript": "~2.4.0 || ~2.5.0 || ~2.6.0 || ~2.7.0 || ~2.8.0 || ~2.9.0",
+    "typescript": ">=2.4.0 < 3.1",
     "webpack": "^4.0.0"
   },
   "devDependencies": {
-    "typescript": "~2.9.2",
```

Comment:
It's still a dev dependency of this package I think. Removing it works because we still have the toplevel one, but keep with workspaces we should have all the dev deps here. That becomes relevant if we have stricter tslint import checks for example.

Reply:
updated